src/features/import/hooks/useColumn.ts

```diff
@@ -10,6 +10,7 @@ import { useAppDispatch, useAppSelector } from 'core/hooks';
 export interface Option {
   value: string;
   label: string;
+  enumChoices?: EnumChoice[];
```

Comment:
Why is this being introduced here?

To me, this feels like it weakens the type (by adding an optional attribute). That can be warranted sometimes, and maybe it is here, but there are several other "special columns" that we've managed to implement without adding special data to this type, such as date columns, organization columns and tag columns.

What is so special about `enum` fields that it must be treated differently than other custom field types here (including dates)?

Reply:
I don't know what I was thinking here really. I removed this and loaded the fields where needed instead using useCustomFields.

Reply:
Removed again in d641dd6319e39bb9351ca7d94631ccc1d93f1371